wled00/wled.h

```diff
@@ -757,6 +759,11 @@ WLED_GLOBAL int8_t spi_sclk  _INIT(SPISCLKPIN);
 WLED_GLOBAL StaticJsonDocument<JSON_BUFFER_SIZE> doc;
 WLED_GLOBAL volatile uint8_t jsonBufferLock _INIT(0);
 
+// global buffers for mqtt responses
+WLED_GLOBAL StaticJsonDocument<JSON_BUFFER_SIZE> mqttResponseDoc;
```

Comment:
Is this really the first time we are sending a response via MQTT?

Reply:
MQTT can receive JSON but is not responding in JSON. I would still re-use existing global buffer if possible.

This is a heavy toll on available RAM.

Reply:
Concur.  There's no need for a second buffer -- the JSON response is being generated after the incoming message is processed, so the buffer can be cleared and re-used without releasing the lock.